splash/qtrender_lua.py

```diff
@@ -487,6 +491,75 @@ def error(error_info):
         ))
         return cmd
 
+    @command(async=True, sets_callback=True, decode_arguments=False)
```

Comment:
You are right. However, I've tried to set it False and nothing has changed (or maybe I don't checked some edge case).